.github/workflows/ci.yml

```diff
@@ -62,19 +65,16 @@ jobs:
       - name: Set Ruby version
         uses: ruby/setup-ruby@v1 # Sets ruby version via `.ruby-version`
 
-      - name: Install bundler
-        run: gem install bundler --version 2.2.3
-
```

Comment:
How do we ensure the correct version of bundler is being used?

Reply:
ruby/setup-ruby handles that by reading the verison from `Gemfile.lock`

![image](https://user-images.githubusercontent.com/1333960/107215798-b6d83180-6a03-11eb-9195-09fff36469f2.png)